tests/acceptance/test_organization_events_v2.py

```diff
@@ -170,3 +184,39 @@ def test_modal_from_errors_view(self, mock_now):
 
             assert self.browser.element_exists_by_test_id("older-event")
             assert self.browser.element_exists_by_test_id("newer-event")
+
+    def test_transactions_query_empty_state(self):
+        with self.feature(FEATURE_NAMES):
+            self.browser.get(self.path + "?" + transactions_query)
+            self.wait_until_loaded()
+            self.browser.snapshot("events-v2 - transactions query - empty state")
+
+    @patch("django.utils.timezone.now")
+    def test_transactions_query(self, mock_now):
+        mock_now.return_value = before_now().replace(tzinfo=pytz.utc)
+        event_data = load_data("transaction")
+        self.store_event(data=event_data, project_id=self.project.id, assert_no_errors=False)
```

Comment:
Why do you need to ignore errors here?

Reply:
@markstory I wasn't entirely aware what `assert_no_errors` does. I'm assuming this should be `True` for transactions.